src/openforms/submissions/templates/submission_logs/rules.html

```diff
@@ -0,0 +1,18 @@
+{% if node.nodeType == 'operation' %}
+    <li class="rule-list"><b> {{node.operator}}: </b> </li>
```

Comment:
`<b>` is not recommended anymore these days. Either use the more semantic `<strong>` or apply the styling with CSS.